starfish/test/full_pipelines/api/test_dartfish.py

```diff
@@ -179,4 +179,4 @@ def test_dartfish_pipeline_cropped_data():
     corrcoef = np.corrcoef(x, y)
     corrcoef = corrcoef[0, 1]
 
-    assert np.round(corrcoef, 5) == -0.00992
+    assert np.round(corrcoef, 5) == 0.04422
```

Comment:
say what?

Reply:
... but now that I go back and look, this is not the correct value for the correlation. Thanks for raising. 

I'll dig in and figure out why this changed.

Reply:
1-off -- Due to the reversion to the previous "`53` identified targets" instead of `54`, we now fail to observe 1 `RELN` count, and as a result, the correlation shifts. This is a direct consequence of the above change.